docs/user-guide/rules/http-cache.md

```diff
@@ -0,0 +1,207 @@
+# HTTP Cache (`http-cache`)
+
+`http-cache` verifies that the page and all its resources follow a good,
+sustainable caching strategy.
+
+## Why is this important?
+
+You can easily improve the performance of your website by having a
+right caching strategy:
+
+* Shorter load times
+* Less bandwidth required
+* Reduce server costs
+* Have a predictable behavior across browsers
+
+Currently about ~50% of resources on the web can't be cached due to
+their configuration:
+
+![Cacheable Resources][maxage0 image]
```

Comment:
This will get outdated over time right? And if so maybe we should just link to http://httparchive.org/about.php#maxage0, that contains this graph?

Reply:
The graph has been pretty consistent for the last year or so. I think the image is nice to show how stale things are and how easy getting perf wins could be.

Reply:
ok.